projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -191,13 +191,21 @@ function_exists( 'wpcom_site_has_feature' ) &&
 	 * Adds the site switcher link if user has more than one site.
 	 */
 	public function add_browse_sites_link() {
-		$site_count = get_user_option( 'wpcom_site_count' );
-		if ( ! $site_count || $site_count < 2 ) {
+		if ( 'wp-admin' === get_option( 'wpcom_admin_interface' ) ) {
+			add_menu_page(
+				esc_attr__( 'All Sites', 'jetpack' ),
+				esc_attr__( 'All Sites', 'jetpack' ),
+				'manage_options',
+				'https://wordpress.com/sites',
+				null,
+				'dashicons-arrow-left-alt2',
+				0
+			);
```

Comment:
this shouldn't work as a replacement for the nav redesign side. As the masterbar module itself is disabled and this code is never loaded.

Reply:
Make sense, I followed your suggested approach and updated the PR